hubble/relay.go

```diff
@@ -261,8 +261,8 @@ func (p *Parameters) RelayPortForwardCommand(ctx context.Context, client k8sHubb
 		"port-forward",
 		"-n", p.Namespace,
 		"svc/hubble-relay",
-		"--address", "0.0.0.0",
-		"--address", "::",
+		"--address", "127.0.0.1",
+		"--address", "::1",
```

Comment:
The patch LGTM but with issue #1093 in mind, do we actually want to bind to the v6 address?

Reply:
I'd suggest to address that in a follow-up PR for #1093 specifically.